odoo/addons/base/models/res_partner.py

```diff
@@ -26,7 +26,7 @@
 WARNING_HELP = 'Selecting the "Warning" option will notify user with the message, Selecting "Blocking Message" will throw an exception with the message and block the flow. The Message has to be written in the next field.'
 
 
-ADDRESS_FIELDS = ('street', 'street2', 'zip', 'city', 'state_id', 'country_id')
+ADDRESS_FIELDS = ('street', 'street2', 'zip', 'city', 'state_id', 'country_id', 'partner_latitude', 'partner_longitude')
```

Comment:
I think that means that "_formatting_address_fields" will now also return lat / lon.
See: https://github.com/odoo/odoo/blob/master/odoo/addons/base/models/res_partner.py#L573

We probably don't want that?

Reply:
Hello @awa-odoo ,

If so, Can't we filter out for "_formatting_address_fields"
by adding this 
```
return [field for field in self._address_fields() if field not in ['partner_latitude', 'partner_longitude']]
```

Thanks

Reply:
Hello.

Yes, I think it would be better to avoid undesired behaviors.